helm/templates/coder.yaml

```diff
@@ -27,6 +27,7 @@ spec:
     metadata:
       labels:
         {{- include "coder.labels" . | nindent 8 }}
+        {{- toYaml .Values.coder.labels | nindent 8 }}
```

Comment:
This should be `podLabels` and have a separate value like you did with annotations